pkg/controller/controlplane/valuesprovider.go

```diff
@@ -687,14 +689,16 @@ func (vp *valuesProvider) GetControlPlaneShootChartValues(ctx context.Context, c
 		return nil, err
 	}
 
-	if validation.ClusterAuditEnabled(&vp.controllerConfig, cpConfig) {
-		if err := vp.deployControlPlaneShootAudittailerCerts(ctx, cp, cluster); err != nil {
-			vp.logger.Error(err, "error deploying audittailer certs")
+	if !extensionscontroller.IsHibernated(cluster) {
+		if validation.ClusterAuditEnabled(&vp.controllerConfig, cpConfig) {
```

Comment:
Does this remove things as well, if the cluster is hibernated?

Reply:
No, it does not remove anything but it wants to talk to the shoot's API server, which is not present during hibernation, and then the reconcile fails.

Reply:
Ah, only certs get deployed here. I thought there was an additional component deployed, which should be scaled-down on hibernation.
So, all good 😄